app/controllers/AdditionalMetadataClosureStatusController.scala

```diff
@@ -38,7 +38,13 @@ class AdditionalMetadataClosureStatusController @Inject() (
     implicit request: Request[AnyContent] =>
       for {
         details <- consignmentService.getConsignmentDetails(consignmentId, request.token.bearerAccessToken)
-        consignment <- consignmentService.getConsignmentFileMetadata(consignmentId, request.token.bearerAccessToken, Some(metadataType), Some(fileIds))
+        consignment <- consignmentService.getConsignmentFileMetadata(
+          consignmentId,
+          request.token.bearerAccessToken,
+          Some(metadataType),
+          Some(fileIds),
+          Some(List(clientSideOriginalFilepath, fileType))
```

Comment:
The list of additional properties is used twice so I would extract it to a private val